tools/scripts/features/frcnn/utils.py

```diff
@@ -0,0 +1,577 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Is this file required for this PR? Also this has lot of unnecessary code. Let's remove them.